### DISCUSSION TITLE
[nodriver] how to Enable Network Request Blocking?

### DISCUSSION THREAD

**Discussion by PacharaXx** [ultrafunkamsterdam/undetected-chromedriver#2010] [Q&A]
![image](https://github.com/user-attachments/assets/43746c68-9b35-403f-9036-297038db54b4)
how can i set a url in Enable Network Request Blocking? cdp?

**Comment by devblack** [comment]
Read docs:
- https://ultrafunkamsterdam.github.io/nodriver/nodriver/cdp/fetch.html#nodriver.cdp.fetch.enable
- https://ultrafunkamsterdam.github.io/nodriver/nodriver/cdp/fetch.html#nodriver.cdp.fetch.RequestPaused
- https://ultrafunkamsterdam.github.io/nodriver/nodriver/cdp/fetch.html#nodriver.cdp.fetch.continue_request
- https://ultrafunkamsterdam.github.io/nodriver/nodriver/cdp/fetch.html#nodriver.cdp.fetch.fail_request
